services/blogs/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@casimir/blogs",
+  "version": "1.0.0",
+  "description": "Server Side Note fetching from HackMD",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "npx esno -r dotenv/config src/index.ts"
+  },
+  "author": "Stevan Najeeb",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.18.2"
```

Comment:
This `express` dependency is where it belongs, so just uninstall the one (and any other rogue packages) sitting in the `package.json` file at the project level.